walkeeper/src/safekeeper.rs

```diff
@@ -378,6 +381,7 @@ where
     }
 
     /// Handle request to append WAL.
+    #[allow(clippy::comparison_chain)]
```

Comment:
What did clippy suggest here instead? Is it ugly?

Reply:
here is the reference with an example https://rust-lang.github.io/rust-clippy/master/index.html#comparison_chain

TLDR, instead of this
```rust
if x > y {
    a()
} else if x < y {
    b()
}
```
use that
```rust
 match x.cmp(&y) {
     Ordering::Greater => a(),
     Ordering::Less => b(),
     Ordering::Equal => {}
 }
```
what do you think about cmp/match option?